src/calibrate-multi-hop/controllers/calibrate.py

```diff
@@ -0,0 +1,55 @@
+import app
+from routes import api
+from flask import Blueprint, request, jsonify
+import logging
+import app
+import json
+import os
+from pymongo import MongoClient
+from models import calibrate as cb
+
+
+calibrate_bp = Blueprint('calibrate_bp', __name__)
+
+
+MONGO_URI = os.getenv('MONGO_URI')
+client = MongoClient(MONGO_URI)
+db = client['airqo_netmanager_staging_airqo']
+col = db['calibration_ratios']
```

Comment:
I sense a mismatch here, the default ENV in the docker file is production but the DB here is hard corded to staging......could we review this? @bobturneruk @Priscilla-AL

Reply:
Good spot - I guess this should be changed @Priscilla-AL?